kernels/volk/volk_64u_byteswap.h

```diff
@@ -242,10 +242,10 @@ static inline void volk_64u_byteswap_neonv8(uint64_t* intsToSwap, unsigned int n
     unsigned int number = 0;
     for (number = 0; number < n4points; ++number) {
         __VOLK_PREFETCH(inputPtr + 8);
-        input = vld2q_u8((uint8_t*)inputPtr);
+        input = vld1q_u8_x2((uint8_t*)inputPtr);
```

Comment:
Our GCC 8 test fails with this:
`/usr/bin/ld: libvolk.so.2.5.2: undefined reference to `vst1q_u8_x2'`

Since newer GCC versions seem to work, We might require smth like an `#ifdef` to switch between those functions.

Reply:
I can do that, but I'll have to figure out what is defined when the `vld1q_u8_x2` instruction exists.

Reply:
That, or require GCC 9 and up. But we need to find some documentation that clearly explains what's going on.